js/about/preferences.js

```diff
@@ -530,6 +617,16 @@ class PaymentsTab extends ImmutableComponent {
       hideParentOverlay={this.props.hideOverlay.bind(this, 'addFunds')} />
   }
 
+  get paymentHistoryContent () {
+    return <PaymentHistory ledgerData={this.props.ledgerData} />
+  }
+
+  get paymentHistoryFooter () {
+    let ledgerData = this.props.ledgerData
+    let nextReconcileDate = formattedDateFromTimestamp(ledgerData.get('reconcileStamp'))
+    return <div className='nextPaymentSubmission'><span>Your next payment submission is {nextReconcileDate}.</span></div>
+  }
+
   btcToCurrencyString (btc) {
```

Comment:
@mrose17 what do you mean? btcPrice was needed to convert `balance`/`unconfirmed` from BTC to local currency amount. if this has changed, please document modified ledger info properties in docs/state.md.